src/App.tsx

```diff
@@ -15,49 +21,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const App: React.FC = () => {
+  const [sortFormat, setSortFormat] = useState<SortType | ''>('');
+  const [doReverse, setDoReverse] = useState<boolean>(false);
+
+  const getPreparedGoods = (
+    groceries: string[],
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    sortFormat: SortType | '',
+    // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
is it possible to it without ` // eslint-disable-next-line @typescript-eslint/no-shadow`?